data-prepper-plugins/s3-source/src/main/java/com/amazon/dataprepper/plugins/source/DateTimeJsonSerializer.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.amazon.dataprepper.plugins.source;
+
+import java.io.IOException;
+import java.util.Date;
+
+import software.amazon.awssdk.utils.DateUtils;
+
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.databind.JsonSerializer;
+import com.fasterxml.jackson.databind.SerializerProvider;
+
+/**
+ * A Jackson serializer for Joda {@code DateTime}s.
+ */
+public final class DateTimeJsonSerializer extends JsonSerializer<Date> {
```

Comment:
Please make this class package protected by removing the `public` modifier. This will help avoid use of this class where it is not intended to be used.